package.json

```diff
@@ -35,7 +36,9 @@
   "engines": {
     "node": ">=4.0.0"
   },
+  "types": "juice.d.ts",
   "dependencies": {
+    "@types/cheerio": "^0.17.31",
```

Comment:
This claims to be definitions for 0.17 but we are on 0.20. Is that just because the API hasn't changed and the comment in the file header is outdated? I'm not clear how these are guaranteed to match.